easybuild/framework/easyconfig/templates.py

```diff
@@ -88,6 +89,8 @@
      'CRAN (contrib) source url'),
     ('FTPGNOME_SOURCE', 'http://ftp.gnome.org/pub/GNOME/sources/%(namelower)s/%(version_major_minor)s',
      'http download for gnome ftp server'),
+    ('GITHUB_SOURCE', 'https://github.com/%(github_account)s/%(namelower)s/archive',
+     'GitHub source url'),
```

Comment:
`"GitHub source URL (requires github_account easyconfig parameter to be specified)"`